src/index.html

```diff
@@ -21,7 +21,8 @@
     passageEndpoint: 'http://localhost:5001',
     environment: 'development',
     version: 'development',
-    ingressBaseDomain: 'k8s.sample.io'
+    ingressBaseDomain: 'k8s.sample.io',
+    installationName: 'development',
```

Comment:
This is the 'default' installation name. It will get overwritten on deployment in the cluster because of the start.sh script which boots up Happa and replaces this value with something present in the ENV variable INSTALLATION_NAME